ios/CoinIconWithProgressBar.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CoinIconWithProgressBar.swift
+//  Rainbow
+//
+//  Created by Alexey Kureev on 06/02/2020.
+//  Copyright © 2020 Facebook. All rights reserved.
```

Comment:
```suggestion
//  Copyright © 2020 Rainbow. All rights reserved.
```

Reply:
Woah! Such a great new feature of Github!